django_prices_openexchangerates/tasks.py

```diff
@@ -49,11 +49,22 @@ def create_conversion_dates():
             continue
         rate = extract_rate(exchange_rates, currency)
         try:
-            conversion_rate, _ = ConversionRate.objects.get_or_create(
+            conversion_rate = _get_or_create_conversionrate(
                 to_currency=currency, rate=rate)
         except Exception:
             logger.exception(
                 'Unable to create ConversionRate',
                 extra={'currency': currency, 'rate': rate})
         else:
             yield conversion_rate
+
+
+def _get_or_create_conversionrate(to_currency, rate):
+    try:
+        conversion_rate = ConversionRate.objects.get(to_currency=to_currency)
+    except ConversionRate.DoesNotExist:
+        conversion_rate = ConversionRate(to_currency=to_currency)
```

Comment:
```suggestion
    conversion_rate = ConversionRate.objects.get_or_create(to_currency=to_currency)
```

Reply:
exception will be caught above
and if ConversionRate really DoesNotExist so next line will raise the error as well